### PR TITLE
Prepare release 2.0.0

### PR DESCRIPTION
**Pull request by fendor**
Update the ChangeLog
Add setup section to README
Bump version number

### CODE REVIEW

**Review by michaelpj — APPROVED** [review]
Haven't been paying attention, but changelog and docs look fine!

**Comment by hasufell** [comment]
This isn't ready for release yet. There's at least one bug to fix, we need to wait for the next ghcup release and then check if we merge https://github.com/haskell/vscode-haskell/pull/547 before that

**Review by hasufell — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -5,6 +5,15 @@
 This extension adds language support for [Haskell](https://haskell.org), powered by the [Haskell Language Server](https://github.com/haskell/haskell-language-server).
 As almost all features are provided by the server you might find interesting read its [documentation](https://haskell-language-server.readthedocs.io).
 
+## Setup
+
+This Extension comes with "batteries"-included and can manage your Haskell Language Server installations for you,
+powered by [GHCup](https://www.haskell.org/ghcup/).
+Installation of [GHCup](https://www.haskell.org/ghcup/) can not happen automatically, so if you want your HLS installations to be
+managed by the Extension, you will have to follow the [installation instructions for GHCup](https://www.haskell.org/ghcup/).
```
Comment (fendor) [inline README.md:13]: @hasufell This is now the first thing new users see, I don't think that's great, but better than nothing

**Comment by fendor** [comment]
Next release is going to be `2.0.1`. Closing this and will re-open for the next release.

**Pull request closed without merging**